app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss

```diff
@@ -14,11 +14,6 @@
  * limitations under the License.
  */
 
-.api-keys-block {
-  max-width: 860px;
```

Comment:
According to the design, the width of each tab is 860 px
Are you really sure that we need to delete it here?
Mb we need to change the width of other tabs?
Could you please discuss it with BA?

Reply:
860px when the screen is 1320px
If the screen is larger then the width of the content also increases. (Victoria confirmed)